apis/openstack-nova/src/test/java/org/jclouds/openstack/nova/v2_0/extensions/AvailabilityZonesApiLiveTest.java

```diff
@@ -34,11 +33,21 @@ public void testListAvailabilityZones() throws Exception {
 
       Optional<? extends AvailabilityZoneApi> availabilityZoneApi = api.getAvailabilityZoneApi("RegionOne");
       if (availabilityZoneApi.isPresent()) {
-         FluentIterable<? extends AvailabilityZone> zones = availabilityZoneApi.get().list();
+         FluentIterable<? extends AvailabilityZone> zones = availabilityZoneApi.get().listAvailabilityZones();
```

Comment:
Make a new test and keep the original test until the deprecated code is removed.